namespaces/live-1.cloud-platform.service.justice.gov.uk/data-eng-rds-dev/resources/rds.tf

```diff
@@ -0,0 +1,67 @@
+
+module "rds" {
+  source               = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.6"
```

Comment:
Hi @green-digger, You need to use the latest release for performance insights, which is 5.9.
https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/releases

Reply:
Made the change to 5.9.